x-pack/test/cases_api_integration/security_and_spaces/tests/common/internal/bulk_get_attachments.ts

```diff
@@ -0,0 +1,437 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { CaseResponse } from '@kbn/cases-plugin/common/api';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+import {
+  postCaseReq,
+  getPostCaseRequest,
+  postCommentUserReq,
+  postCommentAlertReq,
+} from '../../../../common/lib/mock';
+import {
+  deleteAllCaseItems,
+  createCase,
+  createComment,
+  bulkCreateAttachments,
+  ensureSavedObjectIsAuthorized,
+} from '../../../../common/lib/utils';
+import { bulkGetAttachments } from '../../../../common/lib/attachments';
+import {
+  globalRead,
+  noKibanaPrivileges,
+  obsOnly,
+  obsOnlyRead,
+  obsSec,
+  obsSecRead,
+  secOnly,
+  secOnlyRead,
+  superUser,
+} from '../../../../common/lib/authentication/users';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+
+  describe('bulk_get_attachments', () => {
```

Comment:
Can you add a test to check if the API injects the references correctly? One test for each attachment type should be sufficient. You can take a look at how we do it here `x-pack/test/cases_api_integration/security_and_spaces/tests/common/attachments_framework/persistable_state.ts` and here `x-pack/test/cases_api_integration/security_and_spaces/tests/common/attachments_framework/external_references.ts`